src/pages/hyunjunlee/Login/login-hyunjun.js

```diff
@@ -1,9 +1,73 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+import '../../hyunjunlee/Login/lgh.scss';
 
 class LoginHyun extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      IdValue: '',
+      PwValue: '',
```

Comment:
```suggestion
      id: '',
      pw: '',
```
- 변수명은 소문자로 시작하도록 해주세요. 컴포넌트 이름만 대문자로 시작합니다.
- 변수명 간단하게 적어줄 수 있으면 간단히 적어주세요